libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -0,0 +1,85 @@
+/* Standard Includes. */
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "http_client.h"
+
+/**
+ * @brief Default value when pRequestInfo->pPath == NULL.
+ */
+#define HTTP_EMPTY_PATH                    "/"
+#define HTTP_EMPTY_PATH_LEN                ( 1 )
```

Comment:
We use the `sizeof()` utility for length constants to avoid manual errors.
An example is
`#define HTTP_EMPTY_PATH_LEN                (uint8_t) (sizeof(HTTP_EMPTY_PATH) - 1)

Reply:
You sir are a life saver